charts/cluster-api-runtime-extensions-nutanix/templates/deployment.yaml

```
@@ -30,6 +30,9 @@ spec:
         args:
         - --webhook-cert-dir=/runtimehooks-certs/
         - --defaults-namespace=$(POD_NAMESPACE)
+        - --namespacesync-enabled={{ .Values.namespaceSync.enabled }}
+        - --namespacesync-source-namespace={{ .Values.namespaceSync.sourceNamespace | default .Release.Namespace }}
```

As per https://helm.sh/docs/chart_template_guide/function_list/#default

 ```suggestion
        - --namespacesync-source-namespace={{ default .Release.Namespace .Values.namespaceSync.sourceNamespace   }}
```

---

We use the `VALUE | default DEFAULT_VALUE` convention two other places, and the `default DEFAULT_VALUE VALUE` in only one place.  I'd like to fix this in a separate PR, please.

---

Of course you may! 😅